pkgs/by-name/re/re-flex_3/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
+  pname = "reflex";
+  version = "3.2.11";
+
+  src = fetchFromGitHub {
+    owner = "Genivia";
+    repo = "RE-flex";
+    rev = "ref/tags/v${version}";
+    hash = "sha256-ZVa5C1JNmkzz2NhvQEhdVAmT2tBi/PfCEHtWQ/jxf2w=";
+  };
+
+  enableParallelBuilding = true;
+
+  outputs = [
+    "out"
+    "bin"
+    "dev"
+  ];
+
+  meta = {
+    description = ''
+      A high-performance C++ regex library and lexical analyzer generator with Unicode support.
```

```suggestion
      High-performance C++ regex library and lexical analyzer generator with Unicode support
```
See https://nixos.org/manual/nixpkgs/stable/#var-meta-description